README.md

```diff
@@ -1,86 +1,118 @@
 # AMD MIGraphX
 
-AMD's graph optimization engine.
+AMD MIGraphX is AMD's graph inference engine to accelerate model inference on AMD GPUs. AMD MIGraphX can be used by
+installing binaries directly or building from source code.
+
+Note that all the following instructions are based on that ROCm has been installed successfully. ROCm installation
+instructions are explained in the [ROCm installation
+guide](https://rocmdocs.amd.com/en/latest/Installation_Guide/Installation-Guide.html).
+
+## Installing from binaries
+With ROCm installed correctly, MIGraphX binaries can be installed with the following command:
+```
+sudo apt update && sudo apt install -y migraphx
+```
+then the head files and libs are located at `/opt/rocm/include` and `/opt/rocm/lib`, respectively, which can be
+included and linked by adding the corresponding folders to the Makefile.
+
+## Building from source
+
+There are two ways to build the MIGraphX sources. One is installing the dependencies, then using 
+cmake to build the source. The other is using the ROCm build tool [rbuild](https://github.com/RadeonOpenCompute/rbuild).
+In the following, we will first list the dependencies required to build MIGraphX source code, then describe each of the
+two approaches.
+
+### List of dependencies
+The following is a list of dependencies required to build MIGraphX source. This list is also available in the
+requirement files `dev-requirements.txt` and `requirements.txt`.
 
-## Prerequisites
 * [ROCm cmake modules](https://github.com/RadeonOpenCompute/rocm-cmake) **required**
 * [MIOpen](https://github.com/ROCmSoftwarePlatform/MIOpen) for running on the GPU
+* [rocBLAS](https://github.com/ROCmSoftwarePlatform/rocBLAS) for running on the GPU
 * [HIP](https://github.com/ROCm-Developer-Tools/HIP) for running on the GPU
 * [Protobuf](https://github.com/google/protobuf) for reading [onnx](https://github.com/onnx/onnx) files
 * [Half](http://half.sourceforge.net/) - IEEE 754-based half-precision floating point library
 * [pybind11](https://pybind11.readthedocs.io/en/stable/) - for python bindings
 
-## Installing the dependencies
+#### Use cmake to build MIGraphX
 
-Dependencies can be installed using the ROCm build tool [rbuild](https://github.com/RadeonOpenCompute/rbuild).
+In this approach, we need to install the dependencies, configure the cmake, and then build the source.
 
-To install rbuild:
-```
-pip install https://github.com/RadeonOpenCompute/rbuild/archive/master.tar.gz
-```
+##### Installing the dependencies
 
-To build dependencies along with MIGraphX
-```
-rbuild build -d depend --cxx=/opt/rocm/llvm/bin/clang++
-```
-This builds dependencies in the subdirectory named depend and then builds MIGraphX using these dependencies.
+You can manually download and installing the above dependencies one by one. For convience, we provide a shell 
```

Comment:
This seems to imply that you can manually install dependencies or use install_prereqs.sh. This should read

> For convenience, the dependencies can be built automatically with rbuild. If you are on ubuntu 18.04 and have sudo access then you can install the dependencies just like in the docker file by calling ./tools/install_prereqs.sh.

Using rbuild should be emphasized and preferred since its portable.

Reply:
moved rbuild before using cmake..